src/Peering/Peering/Common/Constants.cs

```diff
@@ -34,6 +34,7 @@ public static class Constants
         public const string AzPeeringDirectConnectionObject = "AzPeeringDirectConnectionObject";
         public const string AzPeeringExchangeConnectionObject = "AzPeeringExchangeConnectionObject";
         public const string AzPeering = "AzPeering";
+        public const string AzPeeringReceivedRoutes = "AzPeeringReceivedRoutes";
```

Comment:
powershell cmdlet [guideline](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md) strongly encourage to use singularity in cmdlet name:
>To enhance the user experience, the noun that you choose for a cmdlet name should be singular. For example, use the name Get-Process instead of Get-Processes. It is best to follow this rule for all cmdlet names, even when a cmdlet is likely to act upon more than one item.